templates/loginMerge.html

```diff
@@ -10,20 +10,20 @@ <h3 class="panel-title">{{ match.name }} ({{ match.email }})</h3>
         </div>
         <div class="panel-body form-signin">
           {% for provider in match.providers %}
-            <a class="button zocial {{ provider.name | lower }}" href="{% url 'scopedallaccess-login' provider.name %}">
-              Zaloguj z {{ provider.name | title }}
+            <a class="button zocial {{ provider.name | lower }}" href="{% url 'social:begin' provider %}?next={% url 'finish_merge_verification' %}">
+              Zaloguj z {{ provider | title }}
```

Comment:
We should be really using the "log in with" images to comply with the policies of the OAuth provides - it was changed to links because we were lazy when fixing the non-compliant buttons on the main login page

Reply:
I think we could do more refactoring here, but this is good enough for now